### PR TITLE
New body of request and validations form

### PR DESCRIPTION
**Pull request by betoobernal**
## Summary

* restructured constant data of the technologies.
* add form validations and translations.

### CODE REVIEW

File: src/app/screens/AddMember/i18n.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
-i18next.addResources('es', 'AddMemberToOrganization', {
-  title: 'Agregar miembro a la organización de Wolox',
-  addButton: 'Agregar',
-  labelMenu: 'Agregar miembro a org',
-  userInput: 'Usuario de Github',
-  successMessage: 'El miembro se agregó con éxito.',
-  failedMessage: 'Hubo un error al agregar el miembro a la organización.'
-});
+i18next.addResourceBundle(
```
Comment (FrankIglesias) [inline src/app/screens/AddMember/i18n.js]: Why did you change the i18n format?

File: src/app/screens/AddMemberToTeam/i18n.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 
-i18next.addResources('es', 'AddMemberToTeam', {
-  addButton: 'Agregar',
-  title: 'Agregar miembro(s) al equipo',
-  labelMenu: 'Agregar miembro',
-  userInput: 'Usuario(s) de Github',
-  successMessage: 'El/Los miembro(s) se agregaron con éxito.',
-  failedMessage: 'Hubo un error al agregar el/los miembro(s) al equipo.'
-});
+i18next.addResourceBundle(
```
Comment (FrankIglesias) [inline src/app/screens/AddMemberToTeam/i18n.js]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/app/screens/AddMember/i18n.js (hunk 1 above)
Comment (betoobernal) [reply src/app/screens/AddMember/i18n.js]: The formatter did it. I only changed the addResources for addResourceBundle method. I needed to modify the structure to add nested resources.

File: src/app/screens/RepoCreation/constants.js [hunk 3]
```diff
@@ -1,16 +1,58 @@
...
+    name: 'AngularJS',
+    repositorySufix: 'angular-js'
+  },
+  {
+    name: 'React',
+    repositorySufix: 'react'
+  },
+  {
+    name: 'Vue.js',
+    repositorySufix: 'vue-js'
```
Comment (SKOLZ) [inline src/app/screens/RepoCreation/constants.js]: the sufix should be `vue`

File: src/app/screens/RepoCreation/constants.js [hunk 4]
```diff
@@ -1,16 +1,58 @@
...
-  unity: 'Unity'
-};
+export const TECHNOLOGIES = [
+  {
+    name: 'Rails',
+    repositorySufix: 'rails'
+  },
+  {
+    name: 'NodeJS',
+    repositorySufix: 'node-js'
```
Comment (SKOLZ) [inline src/app/screens/RepoCreation/constants.js]: the sufix shoul;d be `node`

File: src/app/screens/RepoCreation/constants.js [hunk 5]
```diff
@@ -1,16 +1,58 @@
...
+  {
+    name: 'AngularJS',
+    repositorySufix: 'angular-js'
+  },
+  {
+    name: 'React',
+    repositorySufix: 'react'
+  },
+  {
+    name: 'Vue.js',
```
Comment (SKOLZ) [inline src/app/screens/RepoCreation/constants.js]: name: Vue

**Review by SKOLZ — DISMISSED** [review]

**Review by SKOLZ — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]
